### ISSUE TITLE
Win7 needs Zadig driver setup

### ISSUE THREAD

**Issue by nerdralph** [Blinkinlabs/ch554_sdcc#15]
I was getting an error in the device manager for my WCH551 and WCH552 boards.
I tried setting the driver to libusbK using zadig, and still got an error.  libusb-win32 worked.
The CH552 VID/PID is 4348 55E0.

**Comment by cibomahto** [comment]
Hi- this project contains a library and simple framework for the CH55x parts. It sounds like your issue is probably best aimed at the programming tool, can you open an issue there instead? I'll close for now, if I'm not correct please re-open.

**Comment by nerdralph** [comment]
I'm not saying it's a bug in your code, it's a suggestion to improve your "Getting Started" documentation.  There is a cursory mention under the librech551 section about "This program and operation also works on Windows (need Zadig to install driver) and Mac OS.", but nothing more.
If you don't think should go in your docs, you could refer to to ch552tool, which documents the libusb driver setup requirements.
https://github.com/MarsTechHAN/ch552tool

**Comment by cibomahto** [comment]
Ah, thanks for the explanation. I added a section in the Windows instructions that points to LibreCH551 and ch552tool, with a warning about needing Zadig. I never actually tried them on Windows, I switched over to Linux for development as soon as they were available ;-).

If you'd like to write up a quick tutorial for how to set up the driver, or if my explanation was too confusing, I'd be happy to accept changes.
